client/package.json

```diff
@@ -6,6 +6,7 @@
     "@material-ui/core": "^4.11.0",
     "@material-ui/icons": "^4.11.2",
     "axios": "^0.21.0",
+    "lodash": "^4.17.21",
```

Comment:
Is this being used anywhere? If not then it shouldn't be included in the changes.

Reply:
noted !